src/ape/managers/accounts.py

```diff
@@ -20,17 +21,39 @@ def __repr__(self) -> str:
         accounts_str = ", ".join([a.address for a in self.accounts])
         return f"[{accounts_str}]"
 
-    @property
-    def containers(self) -> Dict[str, TestAccountContainerAPI]:
+    def __init__(self):
+        self._init_containers()
+
+    def _init_containers(self):
         containers = {}
+        configs = {}
         account_types = [
             t for t in self.plugin_manager.account_types if issubclass(t[1][1], TestAccountAPI)
         ]
         for plugin_name, (container_type, account_type) in account_types:
             # Pydantic validation won't allow passing None for data_folder/required attr
             containers[plugin_name] = container_type(data_folder="", account_type=account_type)
+            configs[plugin_name] = self.config_manager.get_config(plugin_name)
+        self._containers = containers
+        self._configs = configs
 
-        return containers
+    def _get_configs(self) -> Dict[str, PluginConfig]:
```

Comment:
Why is this method needed?

Reply:
If we dont have this method to check if config is changed or not then one of the tests were failing. Basically there was something like:
`
with temp_config(config):
`
and it was checking for number of test accounts

Reply:
test_custom_num_of_test_accounts_config this test.

Reply:
Hmm, I wonder if we can take an alternative approach here and just not add the new account to `test_accounts`? Basically, those accounts are only the pre-configured genesis accounts, and if you ad-hoc create a new one it is not apart of that set. I think this is how it works if you fetch an account via address e.g. `acct = accounts["myname.eth"]`

Reply:
those accounts are coming from the same approach using generated_dev_accounts and creating TestAccount which has capabilities such as signing messages or transactions. the name test_accounts is also giving people the hint hey dont use these with production accounts. In my opinion, the last approach was just limiting the capabilities of the devs who are using them in tests, so it is perfectly fine for me to use it like this. actually to me, it is better than using context manager approach written in specified test.